content/livesync/connector/index.textile

```diff
@@ -0,0 +1,232 @@
+---
+title: Getting Started
+product: Livesync
+---
+
+The Database Connector uses the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to broadcast application-specific change events from your database to frontend clients via "Ably Channels":/docs/channels. 
+
+When you update the data in your database, you record a change event transactionally in a dedicated "outbox table":https://docs.google.com/document/d/19BJ4oYC3ah4wFH0VS5Qv6qHe3qmRS_PpNLSytTOi9tc/edit#heading=h.r00ywdkts746. The Database Connector detects new changes in the outbox and publishes them as messages on Ably Channels. Using the Models SDK, your client applications subscribe to these change events and update their local state with the new data.
+
+The Database Connector is currently a Docker container image that can run on your own infrastructure. If you want to use an Ably-hosted version, please "reach out":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform.
+
+You can access the Database Connector through a Docker container image, allowing you to run it on your infrastructure in close proximity to your database.
+
+```[sh]
+docker pull ghcr.io/ably-labs/adbc:latest
+```
+
+You can deploy this container image on any cloud infrastructure platform that supports running Docker container images.
+For more information on setting up the required tables in your database, see "Creating the Tables":/livesync/connector/tables.
+
+h2(#local-development). Local Development
+
+In this guide demonstrate the use of "Docker Compose":/https://docs.docker.com/compose/ to spin up a PostgreSQL database and a Database Connector instance on your local machine.
```

Comment:
Should say "In this guide, we will demonstrate the use of..." or "In this guide, we will use..."